### ISSUE TITLE
Rendering issues /w OpenBlocks and EnderIO

### ISSUE THREAD

**Issue by Ad-Blokker** [TinyModularThings/IC2Classic#375]
The cables from IC2C glitch out using tanks from OpenBlocks or EnderIO.
It seems to be a kind of rendering issue, the cables become semi transparant or fully transparant depending on the cable type.

![image](https://user-images.githubusercontent.com/14195799/66567819-519ed280-eb68-11e9-91ed-4619b11b3eaa.png)

Video: https://youtu.be/hewZKipr5GI

p.s. problem persists in the new update of the mod.

**Comment by Speiger** [comment]
Partly a IC2 Bug but a Minecraft game engine limitation. Fixing that would break worlds.

TLDR: All cables share 1 block instance. (Aka 7 cables but 1 Block only used)
The issue is you can only have 1 Transparency state in Minecraft per block.
GlassFiber needs transparancy. So I have toggled it on to be transparent.

And the renderer of minecraft does not like a lot of transparent blocks in combo with TileEntity renderers.

TL:DR. Not easily fixable. I could reduce it down to 1 cable type but not fully fix it, but that would break any existing world.

Not going to happen.

**Comment by Ad-Blokker** [comment]
I keep track of all my issues ;)

Thanks for the response
But is this transparency implementation so limited then? Because with glass fibre cable it also gives a strange look:

![image](https://user-images.githubusercontent.com/14195799/66568968-d4289180-eb6a-11e9-9c57-d97f8f3ccda0.png)

**Comment by OvermindDL1** [comment]
Why is glass fiber rendered as transparent?  It looks like a masked texture, not transparent?

**Comment by Ad-Blokker** [comment]
I see what you mean @Speiger 

This kind of transparency problems I know about. especially because you are looking trough a fluid. 

![image](https://user-images.githubusercontent.com/14195799/66605711-dc0b2480-ebb0-11e9-925b-8a24a5e41724.png)

The my autism just has to life with it, thanks for the responses tho! I only found this because this is a super flat world were everything is open. In my normal world I would hide cables anyways!

The problem with fluids in tanks behind cables is only in full block tanks tho. So other mods that implement tanks that are not full block filling will no glitch out.

**Comment by Ad-Blokker** [comment]
> Why is glass fiber rendered as transparent? It looks like a masked texture, not transparent?

Because they are glass fiber? The name says glass so it makes sense that it is transparant. And it looks nice.
And as I remember correctly in the tekkit versions and other versions of Industrial Craft they are transparent too.

**Comment by OvermindDL1** [comment]
> Because they are glass fiber?

Doesn't matter what they are, look at the texture, it's a masked texture, not a transparent texture.

> And it looks nice.

Changing the rendering stage would not change how it looks.
